pages/1.13/cli/command-reference/dcos-package/index.md

```diff
@@ -3,14 +3,14 @@ layout: layout.pug
 navigationTitle:  dcos package
 title: dcos package
 menuWeight: 12
-excerpt: Installing and managing DC/OS software packages
+excerpt: Installing and managing DC/OS service packages
 
 enterprise: false
 ---
 
 # Description
 
-The `dcos package` commands allow you to install and manage DC/OS software packages.
+The `dcos package` commands allow you to install and manage DC/OS service packages.
```

Comment:
nice one, we should backport this to previous releases

Reply:
This is in already 1.12 (thanks @SarahStegall!). Backporting to 1.11 will not be done in this PR.